proto/v1/event.proto

```diff
@@ -0,0 +1,65 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+option go_package = "./proto";
+package datacommons.v1;
+
+// A collection of events that share some common properties.
+message EventCollection {
+  message GeoLocation {
+    message Point {
+      optional double latitude = 1;
+      optional double longitude = 2;
+    }
+    message Polygon { optional string geo_json = 1; }
+    oneof geo {
+      Point point = 1;
+      Polygon polygon = 2;
+    }
+  }
+
+  message PropVal {
+    optional string prop = 1;
+    repeated string val = 2;
+  }
```

Comment:
is it possible to use a map here? so we can just have:

{
"magnitude": "3.5",
"depth": "Kilometer9.8"
}
this would match our newer v1 APIs. should we also match the triples API here and return provenances for each pv.